.travis.yml

```diff
@@ -9,21 +9,13 @@ env:
     - PA_TEST_ONLINE_INSTALLER=true
   matrix:
     - OTHER_TESTS=true
-    - PRODUCT_TEST_GROUP=0
-    - PRODUCT_TEST_GROUP=1
-    - PRODUCT_TEST_GROUP=2
-    - PRODUCT_TEST_GROUP=3
-    - PRODUCT_TEST_GROUP=4
-    - PRODUCT_TEST_GROUP=5
-    - PRODUCT_TEST_GROUP=6
-    - PRODUCT_TEST_GROUP=7
-    - PRODUCT_TEST_GROUP=8
-    - PRODUCT_TEST_GROUP=9
-    - PRODUCT_TEST_GROUP=10
-    - PRODUCT_TEST_GROUP=11
-    - PRODUCT_TEST_GROUP=12
-    - PRODUCT_TEST_GROUP=13
-    - PRODUCT_TEST_GROUP=14
+    - PRODUCT_TEST_GROUP="tests/product/constants.py tests/product/standalone/presto_installer.py tests/product/prestoadmin_installer.py tests/product/yarn_slider/slider_presto_installer.py tests/product/test_installer.py tests/product/base_product_case.py tests/product/test_script.py tests/product/test_server_install.py"
+    - PRODUCT_TEST_GROUP="tests/product/test_server_uninstall.py tests/product/topology_installer.py tests/product/test_server_upgrade.py tests/product/yarn_slider/slider_installer.py tests/product/test_authentication.py"
+    - PRODUCT_TEST_GROUP="tests/product/yarn_slider/test_server_install.py tests/product/test_control.py"
+    - PRODUCT_TEST_GROUP="tests/product/yarn_slider/test_slider_installation.py tests/product/test_connectors.py"
+    - PRODUCT_TEST_GROUP="tests/product/test_plugin.py tests/product/test_collect.py"
+    - PRODUCT_TEST_GROUP="tests/product/mode_installers.py tests/product/test_status.py tests/product/test_error_handling.py tests/product/standalone/test_installation.py"
+    - PRODUCT_TEST_GROUP="tests/product/yarn_slider/pa_slider_config.py tests/product/test_package_install.py tests/product/test_topology.py tests/product/test_configuration.py"
```

Comment:
this was an older commit to consolidate those product test groups so we didn't have so many jobs running. is the new way the groups are formatted still a problem? i grouped them by the docker images they need to generate before running the test.